README.md

```diff
@@ -45,17 +49,15 @@ See doc/services.drawio for the source of this diagram
 --->
 ![Service Sequence diagram](doc/services.png)
 
-[Palisade Service](palisade-service/README.md) recieves a clients request, returns a unique token and initiates the processing of resources.
-[User Service](user-service/README.md) connects to User Service providers to ensure the user exists.  
-[Resource Service](resource-service/README.md) communicates with the backing stores to ensure the resource exists.
-[Policy Service](policy-service/README.md) checks that policies exist for each resource in a request.  
-[Attribute-Masking Service](attribute-masking-service/README.md) applies policies from the Policy Service against the resources in the request.
-[Topic-Offset Service](topic-offset-service/README.md) listens to kafka topics to inform the Filtered-Resource Service where on the topic (at what offset) the first message for each token appears.
-[Filtered-Resource Service](filtered-resource-service/README.md) readies the resources and later handles passing the data to the client.  
-
-[Data Service](data-service/README.md) retrieves resources from the relevant backing stores. 
-
-[Audit Service](audit-service/README.md) operates in the background providing an audit log for each request. 
+1. [Palisade Service](palisade-service/README.md) receives a clients request, returns a unique token and initiates the processing of resources.
+1. [User Service](user-service/README.md) connects to User Service providers to ensure the user exists.  
+1. [Resource Service](resource-service/README.md) communicates with the backing stores to ensure the resource exists.
```

Comment:
This is a chance to fix these descriptions.  The User Service for example retrieves the information about the User.  Validating is just one step in this process.  Similarly, Resource Service will retrieve all of the requested resources, validating, caching are details we can talk about in the readme for the Resource Service readme.  Most of this can be pulled out of the respective service level readmes.

Reply:
Updated some of the service descriptions